cypress/e2e/spec.cy.js

```diff
@@ -76,31 +76,37 @@ describe('What user sees on home page', () => {
   });
 
   it('should display an error message when the form field is submitted empty, and when the search does not return any watches', () => {
-      cy.get('.submitUserName').click();
-      cy.get('.error-message').should('contain', 'Please enter a name.');
-      cy.get('.submitUserPrice').click();
-      cy.get('.error-message').should('contain', 'Please enter a price.');
-      cy.get('.submitUserType').click();
-      cy.get('.error-message').should('contain', 'Please enter a type.');
-      cy.get('input[name="name"]').type('NoWatchName');
-      cy.get('.submitUserName').click();
-      cy.url().should('eq', 'http://localhost:3001/results');
-      cy.get('.no-results').should('contain', 'There are no watches that meet your criteria, please try again.');
-      cy.visit('http://localhost:3001');
-      cy.get('input[name="price"]').type('20000');
-      cy.get('.submitUserPrice').click();
-      cy.url().should('eq', 'http://localhost:3001/results');
-      cy.get('.no-results').should('contain', 'There are no watches that meet your criteria, please try again.');
-      cy.visit('http://localhost:3001');
-      cy.get('input[name="type"]').type('NoWatchType');
-      cy.get('.submitUserType').click();
-      cy.url().should('eq', 'http://localhost:3001/results');
-      cy.get('.no-results').should('contain', 'There are no watches that meet your criteria, please try again.');
+    cy.get('.submitUserName').click();
```

Comment:
I like that you put all of the testing into one larger test, rather than splitting them up. Cypress can be really slow if you break up the tests into too many components, so this is great!